include/core/CContainerPrinter.h

```diff
@@ -105,7 +105,12 @@ class CElementPrinter<false> {
         return CStringUtils::typeToStringPretty(value);
     }
 
-    //! std::ostringstream implementation.
+    //! std::to_string for float.
+    inline static std::string print_(float value, true_ /*is arithmetic*/) {
+        return std::to_string(value);
+    }
+
+    //! Fallback to std::ostringstream.
     template<typename T>
     inline static std::string print_(const T& value, false_ /*is arithmetic*/) {
```

Comment:
should the comment be /*is not arithmetic*/?

Reply:
In this case, it is `false_ /* is arithmetic */` with the false as well, i.e. the type isn't arithmetic.